examples/image-streams/image-streams-centos7.json

```diff
@@ -22,6 +22,7 @@
               "description": "Build and run Ruby 2.0 applications",
               "iconClass": "icon-ruby",
               "tags": "builder,ruby",
+              "supports": "ruby:2.0,ruby",
```

Comment:
Hmm, I am being explicit, 'cause that's what I understood you said :-) https://github.com/openshift/origin/issues/1878